src/randomInitBoard.c

```diff
@@ -1,4 +1,4 @@
-#include "../include/randomInitBoard.h"
+#include <randomInitBoard.h>

```

Comment:
I would rather keep "" instead of <> for custom libs as the triangle braces used for standard libraries. Still ok